### PR TITLE
Handle dropping capabilities correctly when running as non root user

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@lsm5 @giuseppe @mheon @vrothberg @TomSweeneyRedHat @baude PTAL

File: pkg/specgen/generate/security.go [hunk 1]
```diff
@@ -67,7 +67,8 @@ func securityConfigureGenerator(s *specgen.SpecGenerator, g *generate.Generator,
 		g.SetupPrivileged(true)
 		caplist = capabilities.AllCapabilities()
 	} else {
-		caplist, err = rtc.Capabilities(s.User, s.CapAdd, s.CapDrop)
+
```
Comment (mheon) [inline pkg/specgen/generate/security.go:70]: Nit: extra newline

**Comment by mheon** [comment]
Can we get a test for this?

**Comment by rhatdan** [comment]
@mheon I did add a test for this?

**Comment by mheon** [comment]
@rhatdan Didn't see one when I commented, sorry

**Comment by mheon** [comment]
Tests are red

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/libpod/issues/6645

**Pull request merged**